docs/core/profiles.md

```diff
@@ -139,108 +141,152 @@ And there are others that take on a numeric value, like:
 ```yaml
-speed_factor:
-  get_off_bike: {true: 0.6, false: 1.0}
+speed:
+  - if: get_off_bike
+    multiply by: 0.6
 ```
-which means that for edges with `get_off_bike=true` the speed factor will be `0.6` and otherwise it will be `1.0`.
-You can skip any of these values to retain the default.
 
-For encoded values with numeric values, like `max_width` you use the `<` and `>` operators, like this:
+which means that for edges with `get_off_bike==true` the speed factor will be `0.6`.
+
+For encoded values with numeric values, like `max_width` you should not use "equality" or "inequality" but the
+comparison operators "bigger" `>`, "bigger or equals" `>=`, "smaller" `<` or "smaller or equals" `<=`, e.g.:
 ```yaml
-speed_factor:
-  max_width: {"<2.5": 0.8}
+speed:
+  - if: "max_width < 2.5", multiply by: 0.8
 ``` 
-which means that for all edges with `max_width` smaller than `2.5m` the speed factor is `0.8`.
+which means that for all edges with `max_width` smaller than `2.5m` the speed is multiplied by `0.8`.
 
-In any case values of `speed_factor` have to be in the range `[0,1]` and it is not possible to *increase* the speed for
-edges of certain types. 
+Then there are categories of `string` type. You use them via:
 
-Another way to change the speed is using the `max_speed` section, for example:
 ```yaml
-max_speed:
-  surface: {gravel: 60}
+speed:
+- if: country == "DEU"
+  multiply by: 0
 ```
 
-implies that on all road segments with `surface=gravel` the speed will be at most `60km/h`, regardless of the default 
-speed of this edge or the adjustments made by the `speed_factor` section. Just like with `speed_factor` you can setup
-`max_speed` values for multiple category values and different categories. If multiple rules match for a given edge the
-most restrictive rule will determine the speed (the minimum `max_speed` will be applied). 
-Values for `max_speed` must be in the range `[0,max_vehicle_speed]` where `max_vehicle_speed` is the maximum speed that
-is set for the base vehicle (which you cannot change).
+Another way to change the speed is using `limit to`:
+```yaml
+speed:
+  - if: "surface != GRAVEL", limit to: 60
+```
+
+This implies that on all road segments with no `GRAVEL` value for `surface` the speed will be at most `60km/h`,
+regardless of the default speed of this road segment or the previous `multiply by` statements. Just like
+with `multiply by` you can use multiple category values and different categories in the expressions for `limit to`. 
+If multiple statements match for a given edge the most restrictive statement will determine the speed.
+
+Values for `limit to` must be in the range `[0, max_vehicle_speed]` where `max_vehicle_speed` is the maximum speed that
+is set for the base vehicle, and you cannot change it.
+
+An unconditional statement can be used as the last statement to limit the speed to a certain value under all conditions:
+```yaml
+speed:
+  - if: "true", limit to: 90
+```
```

Comment:
I found this more confusing than helpful. Maybe we should add this at the end of the else section, because this is only needed if one wants the same effect as 'else', but without preceding 'if' statements.

Reply:
Ok. I wanted to mention this explicitly because of the max_speed_fallback we had before

Reply:
Yes sure I just meant I would have found it easier to understand if first the else clause was explained. `if (true)` is just a special case of `if (condition) else` in case `condition==false`.